application/src/main/java/org/opentripplanner/standalone/config/routerequest/RouteRequestConfig.java

```diff
@@ -459,6 +460,16 @@ private static void mapStreetPreferences(NodeAdapter c, StreetPreferences.Builde
               .asInt(dftElevator.hopTime())
           );
       })
+      .withEscalator(escalator -> {
```

Comment:
If you add an example here: https://github.com/opentripplanner/OpenTripPlanner/blob/198ed907d1c5daf48349c205cdb443f0d0c7dfec/application/src/test/resources/standalone/config/router-config.json it will show up in the documentation.